lib/zbc.c

```diff
@@ -38,10 +38,12 @@ static struct zbc_sg_sk_s {
 	enum zbc_sk	sk;
 	const char	*sk_name;
 } zbc_sg_sk_list[] = {
+	{ ZBC_SK_NOT_READY,			"Not-ready"			},
```

Comment:
Can you fix the indentation so that everything aligns nicely ?

Reply:
@damien-lemoal The two lines you pointed out actually show up nicely in both of my text editors. I imagine this has to do with how Github renders tabs. If I "fix" the indentation to make it look right on Github, it won't look right in text editors. What do you want?

Reply:
Here is what I get when I apply your patch:
```
/**
 * Sense key strings.
 */
static struct zbc_sg_sk_s {
        enum zbc_sk     sk;
        const char      *sk_name;
} zbc_sg_sk_list[] = {
        { ZBC_SK_NOT_READY,                     "Not-ready"                     },
        { ZBC_SK_MEDIUM_ERROR,          "Medium-error"          },
        { ZBC_SK_ILLEGAL_REQUEST,       "Illegal-request"       },
        { ZBC_SK_DATA_PROTECT,          "Data-protect"          },
        { ZBC_SK_ABORTED_COMMAND,       "Aborted-command"       },
        { 0,                                            NULL                            }
};
```

So definitely not aligned. Check with a simple plain text editor like vim please.
Anyway, that is fine, I can fix that when applying. Please update your PR with a proper commit message (not just a title) and add your Signed-off-by tag.

Reply:
@damien-lemoal My point was simply that these two lines will always be misaligned for some people. Anyways, I've modified the commit per your instructions.